custom_components/teslafi/model.py

```diff
@@ -241,7 +246,7 @@ def is_defrosting(self) -> bool | None:
         return (
             self.get("is_front_defroster_on") == "1"
             or self.get("is_rear_defroster_on") == "1"
-            or self.get("defrost_mode", "0") != "0"
+            or (self.get("defrost_mode", "0") != "0" and None != self.get("defrost_mode", "0"))
```

Comment:
Can use walrus notation here: 
```suggestion
            or ((d := self.get("defrost_mode", "0")) != "0" and d is not None)
```
Or just "and d"